src/Domain/Entities/Digital/Entry.cs

```diff
@@ -9,6 +9,7 @@ public class Entry : BaseAuditableEntity
     public string Path { get; set; } = null!;
     public Guid? FileId { get; set; }
     public Guid OwnerId { get; set; }
+    public long? Size { get; set; }
```

Comment:
what unit is size? how do I know?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/315/commits/80c9fb7b5afe06e2373c313e26bf54dd9fdf7406